index.src.html

```diff
@@ -2102,6 +2114,27 @@
             |clone|.
     2. Return |clone|.
 
+### AudioDataCopyToOptions ### {#audiodata-copy-to-options}
+
+<xmp class='idl'>
+dictionary AudioDataCopyToOptions {
+  required unsigned long planeNumber;
+  unsigned long frameOffset = 0;
+  unsigned long frameCount;
+};
+</xmp>
+
+: <dfn dict-member for=AudioDataCopyToOptions>planeNumber</dfn>
+:: The number identifying the plane to copy from.
```

Comment:
This is more like an index, not just a number.

Reply:
Done.